src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_Find.java

```diff
@@ -116,6 +117,17 @@ public void onCommand(
                         continue;
                     }
                 }
+                if (excludeOutOfStock) {
+                    if (shop.isSelling()) {
+                        if (shop.getRemainingStock() == 0) {
+                            continue;
+                        }
+                    } else if (shop.isBuying()) {
+                        if (shop.getRemainingSpace() == 0) {
```

Comment:
May be we need a helper method to simply this?
I have saw such statement for 3 times.

Reply:
I Will do it next version